### PR TITLE
Logging Commands + CloudWatchLogs Mock

### PR DESCRIPTION
**Pull request by maikman**

### CODE REVIEW

File: packages/commands-logging/src/write_log.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  CWL = new AWS.CloudWatchLogs({ region: process.env.AWS_REGION })
+
+  executeCloud = this.exec.bind(this, this.CWL)
+  executePure = this.exec.bind(this, CWLSandbox as any)
+  executeConnected = this.executeCloud
+  executeEdge = this.exec.bind(this, new CWLMockFS('/tmp/.data') as any)
+
+  constructor(params: WriteLogRequest) {
+    super()
+    const events = _.map(params.logs, param => {
```
Comment (Aduril) [inline packages/commands-logging/src/write_log.ts]: wieso lodash map?

Comment (Aduril) [reply packages/commands-logging/src/write_log.ts]:
```javascript
const events = params.logs.map(mapLogToEvent)
...
function mapToEvent(log:Log) {
  return { 
    message: JSON.stringify(log.message),
    timestamp: log.timestamp || moment().unix()
  }
}
```

File: packages/commands-logging/src/write_log.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+}
+
+/**
+ * Command to write log object to cloudwatch
+ */
+export class WriteLog extends Command<
+  PromiseResult<PutLogEventsResponse, AWS.AWSError>
+> {
+  params: PutLogEventsRequest
+  CWL = new AWS.CloudWatchLogs({ region: process.env.AWS_REGION })
```
Comment (Harper04) [inline packages/commands-logging/src/write_log.ts]: you're the first to use aws region
not sure if i like process.env, it should be there on a lambda; it is not on edge but should be no problem for now (will be initialized anyway)

Comment (maikman) [reply packages/commands-logging/src/write_log.ts]: you can't initialize it without region. But you are right, it is undefined on edge. should i hardcode it for now and solve it with the config package later?

**Pull request merged**